mappings/net/minecraft/nbt/NbtOps.mapping

```diff
@@ -18,4 +18,58 @@ CLASS net/minecraft/class_2509 net/minecraft/nbt/NbtOps
 		COMMENT with an ordered list of values in serialization). In fact, since
 		COMMENT Minecraft NBT lists can only contain elements of the same type, this op
 		COMMENT cannot compress maps.</p>
+	METHOD convertTo (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 2 element
+	METHOD createBoolean (Z)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createByte (B)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createDouble (D)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createFloat (F)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createInt (I)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createLong (J)Ljava/lang/Object;
+		ARG 1 value
+	METHOD createNumeric (Ljava/lang/Number;)Ljava/lang/Object;
```

Comment:
Should this match up with `AbstractNbtNumber`'s `getNumber` method?

Reply:
`AbstractNbtNumber#getNumber` could be renamed to `AbstractNbtNumber#getNumeric`

but NbtOps cannot be renamed as it's an overriden method.